@kiva/kv-components/vue/KvMap.vue

```diff
@@ -408,6 +505,96 @@ export default {
 				}, timeout);
 			});
 		},
+		getCountriesData() {
+			const countriesFeatures = countriesBorders.features ?? [];
+
+			countriesFeatures.forEach((country, index) => {
+				const countryData = this.countriesData.find((data) => data.isoCode === country.properties.ISO_A2);
+				if (countryData) {
+					countriesFeatures[index].lenderLoans = countryData.value;
+					countriesFeatures[index].numLoansFundraising = countryData.numLoansFundraising;
+				}
+			});
+
+			return countriesBorders;
+		},
+		countryStyle(feature) {
+			return {
+				color: kvTokensPrimitives.colors.white,
+				fillColor: this.getCountryColor(feature.lenderLoans),
+				weight: 1,
+				fillOpacity: 1,
+			};
+		},
+		getCountryColor(lenderLoans) {
```

Comment:
Could this method use some unit tests as well? I know there's some randomness to the colors, so I understand if a unit test might not be as reliable.

Reply:
I added one for default color.